src/pages/content/index.tsx

```diff
@@ -69,8 +69,26 @@ async function initialize() {
 
         if (pageData.type === "root") {
             resetRootStyles()
-
-            throw new Error("root page")
+            
+            rootContainer.setAttribute("id", "__better_studres_theme_root");
+            rootContainer.style.overflowY = "scroll";
+        
+            rootContainer.innerHTML = "";
+        
+            const styleElements = document.querySelectorAll('style, link[rel="stylesheet"]');
+            styleElements.forEach(el => el.remove());
+        
+            rootContainer.style.margin = "0";
+            rootContainer.style.padding = "0";
```

Comment:
maybe just add class `_tailwind_preflight_reset` to reset styles?

Reply:
Tried, but didn't work properly.